docs/src/views/Theming.tsx

```diff
@@ -87,7 +99,7 @@ export default () => (
       render={() => (
         <>
           <Icon name="user" circular />
-          <Icon name="user" circular variables={{ color: 'cornflowerblue' }} />
+          <Icon name="user" circular styles={{ color: 'cornflowerblue' }} />
```

Comment:
while it makes the code example correct, unfortunately, it makes the intent of the paragraph to be invalid, as here we suggesting to use variables:

> You can define component variables on a single instance of a component.

But then we are providing a code example that is about `styles` customization.

Reply:
Let's try to use colors from the color palette, to keep the consistency across the API examples.